src/components/Trend/__test__/Trend.test.js

```diff
@@ -79,4 +79,25 @@ describe('Trend', () => {
     expect(lastStopProps.offset).toEqual(1);
     expect(lastStopProps.stopColor).toEqual('red');
   });
+
+  it('handles an array of value objects', () => {
+    // I don't really know the best way to test this, beyond checking that
+    // it doesn't blow up.
```

Comment:
One way to test that value objects work is checking that passing `data={[0, 10, 5]}` and `data={[{ value: 0 }, { value: 10 }, { value: 5 }]}` result in identically rendered components